code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -51,6 +52,8 @@
 	desc = "An M4A3 Colt Service Pistol, the standard issue sidearm of the TerraGov Marine Corps. Uses 9mm pistol rounds."
 	icon_state = "m4a3"
 	item_state = "m4a3"
+	caliber = "9x19mm Parabellum" //codex
```

Comment:
Please understand the consequences of changing ALL of the calibers. This affects actual in-game functionality (they're just strings)
Add another variable if you absolutely must.

Reply:
This is a new variable. Caliber is attached to the bullet, and the selected caliber for a weapon is attached to the magazines they can load. Before this PR, weapons never had a specific caliber.

Reply:
I know this because weapon caliber is different than ammo caliber (because of strings) yet it still fires and damages objects.